web/src/screens/settings/Releases.tsx

```diff
@@ -18,10 +18,10 @@ function ReleaseSettings() {
   const queryClient = useQueryClient();
 
   const deleteMutation = useMutation({
-    mutationFn:  APIClient.release.delete,
+    mutationFn: APIClient.release.delete,
```

Comment:
I saw you did this in previous PRs, but was too lazy to comment, but I'll write it just so you know: You should wrap your queryFn's/mutationFn's in a wrapper function because you will get argument tainting that /one/ time and you'll be scratching your head for a bit until you figure out what's the issue. TS will surely help, but don't rely solely on it.

Put in another way `mutationFn: (variables: TVariables) => Promise<TData>`, the `variables` parameter will get passed on `APIClient.release.delete` and if that function takes an argument, you might not get what you want.

Not an issue in this case, but definitely not a great practice.